docs/content/howto/open-any-file.md

```diff
@@ -63,8 +67,8 @@ Check out our examples for [C++](https://github.com/rerun-io/rerun/tree/main/exa
 
 ### Custom data-loaders
```

Comment:
Maybe that subtitle should be explicitly about rust data loaders. I'd say at least from a user's pov the other ways of adding file formats are also custom data loaders, just not in the `trait` sense 🤔